cmd/faToPredictSet/faToPredictSet.go

```diff
@@ -0,0 +1,77 @@
+// Command Group: Deep Learning
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+)
+
+func faToPredictSet(s Settings) {
+	var err error
+	var i, j int
+	records := fasta.Read(s.InFile)
+	out := fileio.EasyCreate(s.OutFile)
+	var currFa fasta.Fasta
+	var lineToWrite string
+
+	for i = range records {
+		for j = 0; j < len(records[i].Seq)-s.WindowSize; j += s.Stride {
+			currFa = fasta.Extract(records[i], j, j+s.WindowSize, fmt.Sprintf("%s:%v-%v", records[i].Name, j, j+s.WindowSize))
+			lineToWrite = fmt.Sprintf("%s\t%s\n", currFa.Name, dna.BasesToString(currFa.Seq))
+			_, err = fmt.Fprintf(out, lineToWrite)
+			exception.PanicOnErr(err)
+		}
+	}
+
+	err = out.Close()
+	exception.PanicOnErr(err)
+}
+
+func usage() {
+	fmt.Print(
+		"faToPredictSet - Make deep learning prediction CSV files from input fasta format data.\n" +
```

Comment:
Now I understand it's actually supposed to be a tab-delimited (tsv) file

Reply:
fixed in next commit